scripts/build.js

```diff
@@ -128,11 +129,18 @@ function buildFileFor( file, silent, environment ) {
  */
 function buildPackage( packagePath ) {
 	const srcDir = path.resolve( packagePath, SRC_DIR );
+	const packageFile = packagePath + '/package.json';
 	const files = glob.sync( srcDir + '/**/*.js', { nodir: true } )
 		.filter( file => ! /\.test\.js/.test( file ) );
 
 	process.stdout.write( `${ path.basename( packagePath ) }\n` );
 
+	const config = require( packageFile );
+	if ( ! config.publishConfig || 'public' !== config.publishConfig.access ) {
+		process.stdout.write( `${ ERROR } package.json must include "publishConfig": { "access": "public" }\n` );
+		process.exitCode = 1;
```

Comment:
I think with this usage, the script will continue to run, but exit with a code of `1`? vs. [`process.exit( 1 )`](https://nodejs.org/api/process.html#process_process_exit_code) which terminates immediately.

Reply:
Yah, I figured it was fine to let it run to completion, so you see build errors for all modules, if there are multiple.